gtsam_unstable/geometry/Similarity3.cpp

```diff
@@ -85,10 +85,110 @@ Point3 Similarity3::transformFrom(const Point3& p, //
   return s_ * q;
 }
 
+Pose3 Similarity3::transformFrom(const Pose3& T) const {
+  Rot3 R = R_.compose(T.rotation());
+  Point3 t = Point3(s_ * (R_.matrix() * T.translation().vector() + t_.vector()));
```

Comment:
This might fail if `GTSAM_TYPEDEF_POINTS_TO_VECTORS` is enabled.

Reply:
Yes, you are right it fails. I can fix this.